### PR TITLE
add features overview to the README and perform copy editing

### PR DESCRIPTION
**Pull request by tbm**
While the manual contains a detailed description of all features,
it makes sense to add an overview of what is supported.

### CODE REVIEW

File: docs/manual.md [hunk 1]
```diff
@@ -138,7 +145,7 @@ automatically:
    upper case letters (`Assets:test` becomes `Assets:Test`)
 3. Strips accents and umlauts because they are currently not
    supported in beancount ([issue
-   171](https://bitbucket.org/blais/beancount/issues/171)).
+   171](https://bitbucket.org/blais/beancount/issues/161)).
```
Comment (zacchiro) [inline docs/manual.md]: This looks wrong, either it's 161 or 171… but it should be the same in anchor and link ;)

File: README.md [hunk 2]
```diff
@@ -29,28 +29,93 @@ the converted data to `stdout`.  You can run ledger2beancount like this:
...
+Please see [the manual](docs/manual.md#installation) for dependencies and
+installation instructions.
+
+
+Features
+--------
+
+The majority of ledger features are supported by ledger2beancount.  Here
+is an overview of fully supported, partly supported and unsupported features.
+Please refer to the [manual for more details](docs/manual.md#features).
```
Comment (zacchiro) [inline README.md]: While we're nitpicking, I'd either hyperlink only "manual" or the whole sentence starting to "refer to ..."

File: docs/manual.md (hunk 1 above)
Comment (tbm) [reply docs/manual.md]: Thanks!  Glad you're reviewing my changes.

File: README.md (hunk 2 above)
Comment (tbm) [reply README.md]: Ok.

**Pull request merged**